README.md

```diff
@@ -1,75 +1,39 @@
-
-This workflow is set to be run once per time, abolishing concurrent runs.
-
-#### Requirements
-
-This workflow template leverages on managed identities to authenticate with Azure. Managed identities can be easily created through the module `azure_federated_identity_with_github` available in this repository.
-
-Terraform definitions are intended to work for an environment in a specific region. Each pair environment/region is a Terraform project on its own and they will be located in the `<env>/<region>` subfolder. Every automation will expect resources to be in such folders.
-
-## Terraform Modules
-
-### GitHub SelfHosted Runner on Container App job
-
-This module helps to create the necessary infrastructure to run GitHub Actions on a self-hosted agent. Generally, this scenario pertains to infrastructure pipelines requiring access to private resources such as AKS and Storage Accounts. If there aren't private resources, please use GitHub managed runners.
-
-This module creates a Container App Job on an existing Container App Environment. It requires your repository name as input, in order to create a trigger on the GitHub Actions status of that repository. Moreover, it has a dependencies on a KeyVault where it creates a read-only access policy to access secrets.
-
-N.B. If you are adding a new Container App Job, it is likely that your repository must be added to GITHUB PAT access scope. Ask for this one-time operation to `engineering-team-cloud-eng` team.
+<div align="center">
+
+<img src="./assets/pagopa-logo.jpg" width="100" alt="PagoPA icon">
+
+# DevEx Initiative: Transforming the Development Experience at PagoPA
+
+![Static Badge](https://img.shields.io/badge/getting%20started-blue?logo=docusaurus&label=doc&cacheSeconds=5000&link=https%3A%2F%2Fpagopa.github.io%2Fdx%2Fdocs%2F)
+[![Shield: Doc](https://img.shields.io/badge/contributions-welcome-red)](#Contributing)
+![Static Badge](https://img.shields.io/badge/latest%20news-blue?logo=docusaurus&label=blog&cacheSeconds=5000&link=https%3A%2F%2Fpagopa.github.io%2Fdx%2Fblog%2F)
+
+Imagine being able to release the first API for a new digital service into production in minutes instead of weeks, having fewer decisions to make, less code to interpret and maintain, onboarding new team members with zero downtime: this is the goal we set for ourselves with the Developer Experience (DevEx) initiative.
+
+At the heart of the Engineering Area, a group of Senior, Cloud, and Staff Engineers has decided to tackle the daily challenges that slow down our work. We're here to break down barriers, simplify processes, and make software development smoother and more rewarding for everyone.
+
+**Terraform Download Stats**
+
+![Terraform Module Downloads](https://img.shields.io/terraform/module/dm/pagopa-dx/azure-role-assignments/azurerm?label=azure-role-assignments&cacheSeconds=5000&link=https%3A%2F%2Fregistry.terraform.io%2Fmodules%2Fpagopa-dx%2Fazure-role-assignments%2Fazurerm%2Flatest&logo=terraform)
```

Comment:
suggestion: this list would be hard to maintain in sync with codebase. a link to the terraform registry is enough (and better) imho.

Reply:
Imho it's good to show some statistics, to make clear the project is active and used. Unfortunately there is no way to get the sum of all downloads, however these numbers could be useful to us as well

Reply:
question: how do you plan to maintain in sync with modules directory? maybe we should think about some readme generator from a template and a simple bash script that reads the list from the hashicorp registry to compose the final output.